cypress/page/index.ts

```diff
@@ -20,3 +20,5 @@ export { OrderConfirmationPage } from "./orderconfirmation.page";
 export { DressesListPage } from "./dresseslist.page";
 // eslint-disable-next-line import/no-unresolved
 export { PersonalFormPage } from "./personalForm.page";
+// eslint-disable-next-line import/no-unresolved
```

Comment:
It shouldn't throw that error, maybe there's additional config so it resolves these modules